### ISSUE TITLE
Add Text transcript content area in Prismic and bring into Gatsby

### ISSUE THREAD

**Issue by darthvadur** [all-purpose/ap-website-v3#46]
* Need design input
* The link to the transcript should be the first link after the video
* collapsible region might work well here with vertical scrolling area.
